.github/workflows/main.yaml

```diff
@@ -3,7 +3,7 @@ name: CI
 on: ["push"]
 
 jobs:
-  build-and-push:
+  test:
```

Comment:
https://github.com/cognitedata/terraform/blob/master/github/apps/main.tf#L1433 will need to be updated, once manually in repo settings is ok but to avoid terraform applies to revert it need to persist in terraform too

Reply:
ah that sucks, we could rename the pipeline back, but I feel like it makes more sense to change it in terraform too right?

Reply:
yeah, it no longer pushes, but for a quick iteration we can keep old name and rename separately

Reply:
let's do that